### PR TITLE
[zh] Sync concepts pages for extend-kubernetes\operator.md

### PR DESCRIPTION
**Pull request by ydFu**
* Sync with english version in 'Update operator page to list tools in alphabetical order (#26676)'

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/operator.md [hunk 1]
```diff
@@ -185,57 +185,63 @@ kubectl edit SampleDB/example-database # 手动修改某些配置
...
 * 详细了解[定制资源](/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
 * 在 [OperatorHub.io](https://operatorhub.io/) 上找到现成的、适合你的 Operator
-* 借助已有的工具来编写你自己的 Operator，例如：
-  * [KUDO](https://kudo.dev/) (Kubernetes 通用声明式 Operator)
-  * [kubebuilder](https://book.kubebuilder.io/)
-  * [Metacontroller](https://metacontroller.app/)，可与 Webhook 结合使用，以实现自己的功能。
-  * [Operator Framework](https://operatorframework.io)
 * [发布](https://operatorhub.io/)你的 Operator，让别人也可以使用
-* 阅读 [CoreOS 原文](https://coreos.com/blog/introducing-operators.html)，其介绍了 Operator 介绍
+* 阅读[CoreOS原始文章](https://web.archive.org/web/20170129131616/https://coreos.com/blog/introducing-operators.html)，它介绍了Operator模式（这是一个存档版本）的原始文章）。
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/operator.md:244]:
```suggestion
* 阅读 [CoreOS 原始文章](https://web.archive.org/web/20170129131616/https://coreos.com/blog/introducing-operators.html)，它介绍了 Operator 模式（这是一个存档版本）的原始文章）。
```

Comment (ydFu) [reply content/zh/docs/concepts/extend-kubernetes/operator.md]: Thank you for suggesting me.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**